skipper/runner.py

```diff
@@ -60,6 +60,9 @@ def _run_nested(fqdn_image, environment, command, interactive, name, net, publis
     else:
         cmd += ['--rm']
 
+    if not os.environ.get("NO_ULIMIT", False):
+        cmd += ['--ulimit', "nofile=1024000:1024000"]
```

Comment:
1. why different?
2. maybe:
```suggestion
    DEFAULT_SKIPPER_ULIMIT = os.environ.get('SKIPPER_ULIMITS', 'nofile=65536:65536')
    ....
    ulimits = DEFAULT_SKIPPER_ULIMIT.split(',')  # format: x=y,a=b
    for limit in unlimits.split(','):
        cmd += ['--ulimit', limit]
```

Reply:
@ofir-amir, why split(",") twice?
also if we support SKIPPER_ULIMIT in env we should support it with a flag as well.